doc/source/train/common/torch-configure-run.rst

```diff
@@ -1,10 +1,10 @@
-Configure scale and GPUs
-------------------------
+Configure scale and resources (CPU, GPU, and other accelerators)
+----------------------------------------------------------------
 
 Outside of your training function, create a :class:`~ray.train.ScalingConfig` object to configure:
 
 1. :class:`num_workers <ray.train.ScalingConfig>` - The number of distributed training worker processes.
-2. :class:`use_gpu <ray.train.ScalingConfig>` - Whether each worker should use a GPU (or CPU).
+2. :class:`use_gpu <ray.train.ScalingConfig>` - Whether each worker should use a CPU, GPU, or :ref:`other accelerators <using-other-accelerators>`.
```

Comment:
The `use_gpu` flag itself does control other resources.

We can probably revert back to the original text here. Optionally, we can add a third bullet point for `resources_per_worker`, though we should caveat that it's optional/advanced.

Also worth noting that we reference the user guide immediately below, which has a reference to other resources in its title if the user needs that configurability!
<img width="919" alt="image" src="https://github.com/user-attachments/assets/17f1bb2c-3a34-471b-abb8-583154d44ce5" />